src/bin/main.rs

```diff
@@ -139,6 +139,7 @@ mod tests {
         cl.send(&serde_json::to_string(&run_req).unwrap());
 
         cl.recv_resp(); // LogMessage notification
+        cl.recv_resp(); // TaskFinished notification
         let server_resp: Response = from_str(&cl.recv_resp()).unwrap();
         assert_eq!(
             serde_json::to_string(&server_resp).unwrap(),
```

Comment:
czy są jakieś przeciwwskazania żeby do tego użyć insta?

Reply:
Kasia w innym PR i tak zmienia ten plik. Dodatkowo, gdybyśmy używali insta, to powstałyby mało czytelne stringi